### PR TITLE
[OSDEV-836] Update  kickoff instructions in the readme

### PR DESCRIPTION
**Pull request by killev**
Update kick-off instructions to make it possible to start project not asking anyone else

### CODE REVIEW

**Comment by VadimKovalenkoSNF** [comment]
I think you should remove any leftovers about `docker-compose.dd.yml` in the scope of this ticket. Here is one of them: https://github.com/opensupplyhub/open-supply-hub/blob/main/scripts/server#L24

File: README.md [hunk 1]
```diff
@@ -56,59 +53,44 @@ See [Getting Started with Google Maps Platform](https://developers.google.com/ma
...
 
 1. The project containers must be running locally.
-2. Increase Django timeout to have enough time for rebuilding Gazeteer. (docker-compose.yml -> django -> command -> - "--timeout=90" or /django/-> Dockerfile -> CMD -> "--timeout=60", \)
-3. Download prod dump file
+2. Download prod dump file
+3. Place it in `[path-to-project]/dumps/` folder
 4. Then run in the terminal of your machine
 ```
-docker exec -i opensupplyhub-database-1 pg_restore --verbose --clean --no-acl --no-owner -d openapparelregistry -U openapparelregistry < /path/on/your/machine/latest_prod.dump
+docker compose run database --rm --entrypoint pg_restore -d openapparelregistry ./dumps/db.dump -c -U openapparelregistry
```
Comment (vladsha-dev) [inline README.md:85]: I verified. This command doesn't work. I would leave the previous command but apply [changes](https://spdfn.slack.com/archives/C04RGFZC1GW/p1707470972791779?thread_ts=1707462211.802249&cid=C04RGFZC1GW) that were proposed by Vadim.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (killev) [reply README.md:85]: @vladsha-dev If this doesn't work it needs to be fixed. 

Also could you please mention changes here. Please note that this is public repository and not everyone has access to the slack

File: README.md [hunk 2]
```diff
@@ -56,59 +53,44 @@ See [Getting Started with Google Maps Platform](https://developers.google.com/ma
...
-### Development & Fill the Database
-
-You will need to execute migration process, run
-```
-scripts/update
-```
-then reset database and repopulate with fixture data that including parse & geocode processing, run
-```
-scripts/reset_database
+### Kick-off & start local development
```
Comment (vladsha-dev) [inline README.md:56]: With the new approach to setting up the project, we ended up with 350 seeded facilities after installation. However, with [the approach Roman suggested](https://spdfn.slack.com/archives/C04RGFZC1GW/p1707463361257669?thread_ts=1707462211.802249&cid=C04RGFZC1GW) previously, I consistently obtained 927 facilities. We have obtained 927 seeded facilities after the each initial installation for one year.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vladsha-dev) [reply README.md:85]: Just change the name of the container to the correct one in the initial command.

Comment (killev) [reply README.md:85]: > Just change the name of the container to the actual one in the initial command.

@vladsha-dev how exactly. Name of container is can be different?

Comment (vladsha-dev) [reply README.md:85]: Apologies for any potential confusion. What I meant to do was:
docker exec -i **open-supply-hub-database-1** pg_restore --verbose --clean --no-acl --no-owner -d openapparelregistry -U openapparelregistry < /path/on/your/machine/latest_prod.dump

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**